code.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import xarray as xr
+
+lat = 43.07
+lon = 360-89.40
+
+# ERA5
+dset = xr.open_dataset('https://ds.nccs.nasa.gov/thredds/dodsC/bypass/CREATE-IP/reanalysis/ECMWF/IFS-Cy41r2/ERA5/mon/atmos/tas.ncml')
+era5 = dset['tas'].sel(lat=lat, lon=lon, method='nearest').compute() - 273.14
+
+# CMIP6
```

Comment:
This block of code is all commented out.  Not relevant for this script, so please remove it.

Reply:
Thanks.  Removed this.